code/script.js

```diff
@@ -29,23 +83,358 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+ 
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
+  
   showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+
+}
+
+
+btn.addEventListener("click" , (e) => {
+  e.preventDefault();
+
+  const enterName = nameInput.value.trim();
+  
+  if(enterName !== ""){
+    message = nameInput.value;
+    showMessage(enterName , 'user');
+    clearInput();
+    setTimeout(() => {
+      showMessage(`Nice to meet you ${enterName}.what type of food would you like to order?` , 'bot')
```

Comment:
You're so focused on the code (which I understand, and you should be!), if you spent two minutes more on the statements, it would be flawless! It's easy corrections that you know, you're so careful with the code, it feels sloppy when the text is not as well done! You have however focused on the important things when it comes to the project!